src/requests/spec.js

```diff
@@ -23,15 +23,16 @@ const getSpec = (specType, pathParams, queryParams, accept) => {
     })
   }
 
-const putSpec = (specType, pathParams, body) => {
-    const { SWAGGERHUB_URL, SWAGGERHUB_API_KEY } = config.getConfig()
-    return http({
-        url: [SWAGGERHUB_URL, specType, ...pathParams],
-        auth: SWAGGERHUB_API_KEY,
-        contentType: 'json',
-        method: 'PUT',
-        body
-    })
+const putSpec = (specType, pathParams, body, queryParams) => {
+  const { SWAGGERHUB_URL, SWAGGERHUB_API_KEY } = config.getConfig()
+  return http({
+    url: [SWAGGERHUB_URL, specType, ...pathParams],
+    auth: SWAGGERHUB_API_KEY,
+    contentType: 'json',
+    method: 'PUT',
+    body,
+    query: queryParams
```

Comment:
change the fourth param to `query` so you can do es6 assignment and not bother with `query: queryParams`

Reply:
done